independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/InjectionPointInfo.java

```diff
@@ -229,20 +236,35 @@ TypeAndQualifiers getTypeAndQualifiers() {
     }
 
     /**
+     * This method is deprecated and will be removed at some point after Quarkus 3.12.
```

Comment:
This doesn't give much space, does it? I would recommend to wait at least 3 releases, i.e.  `at some point after Quarkus 3.15`.

Reply:
Well, it deliberately says `some point after`, I don't think we need to purge it immediately.
But yea, we can mention 3.15

Reply:
For sure, but it's always a good idea to inform the user how much time is left... `at some point after Quarkus 3.12` means that it could be removed in any version after 3.12 so from a user POV the only safe thing to do is to change the code immediately. Which might be a bit stressful ;-)